YAML and JSON export in Konnect (#6348)
Kong/docs.konghq.com: app/konnect/updates.md

services. [Try it today!](https://cloud.konghq.com/quick-start)

## November 2023

**Support for exporting configurations in YAML and JSON**

> : {{site.konnect_short_name}} users can now export Gateway Manager objects like services, routes, plugins to YAML or JSON and import them into APIOps tooling like DecK.

Reviewer: ```suggestion
: {{site.konnect_short_name}} users can now export Gateway Manager objects like services, routes, plugins to YAML or JSON and manage them with APIOps tooling like decK.
```
"Import them into decK" is not really a sentence that makes sense. You can't really import things into decK, it's a CLI tool that imports/exports stuff for you.

> 

Reviewer: It might be useful to add a link to the reference for entities that decK can manage, since you provided a short sample list: https://docs.konghq.com/deck/latest/reference/entities/
